src/components/common/TfMain.tsx

```diff
@@ -2,23 +2,54 @@ import React from 'react';
 import { makeStyles } from '@mui/styles';
 import { Theme } from '@mui/material';
 
+import useDeviceInfo from 'src/utils/useDeviceInfo';
+
+interface StyleProps {
+  backgroundImageVisible?: boolean;
+  isMobile?: boolean;
+}
+
 const useStyles = makeStyles((theme: Theme) => ({
   main: {
     // TODO: uncomment this line when updating to the new green color theme
     // background: 'linear-gradient(180deg, #FBF9F9 0%, #EFF5EF 100%)',
     minHeight: '100vh',
     display: 'flex',
     flexDirection: 'column',
-    padding: '32px',
+    padding: (props: StyleProps) => (props.isMobile ? '32px 24px' : '32px'),
+    [theme.breakpoints.down('xl')]: {
+      background: (props: StyleProps) =>
+        !props.backgroundImageVisible
+          ? 'initial'
+          : 'url(/assets/home-bg-right-layer-z4.svg) no-repeat 753px 100%/auto 285px, ' +
+            'url(/assets/home-bg-left-layer-z4.svg) no-repeat 0 100%/auto 295px, ' +
+            'url(/assets/home-bg-water-z2.svg) repeat-x 0 100%/auto 180px, ' +
+            'url(/assets/home-bg-left-z4.svg) no-repeat 0 100%/auto 295px, ' +
+            'url(/assets/home-bg-right-z3.svg) no-repeat 911px 100%/auto 400px',
+      backgroundAttachment: 'fixed !important',
+    },
+    [theme.breakpoints.up('xl')]: {
+      background: (props: StyleProps) =>
+        !props.backgroundImageVisible
+          ? 'initial'
+          : 'url(/assets/home-bg-right-layer-z4.svg) no-repeat 100% 100%/auto 285px, ' +
+            'url(/assets/home-bg-left-layer-z4.svg) no-repeat 0 100%/auto 295px, ' +
+            'url(/assets/home-bg-water-z2.svg) repeat-x 0 100%/auto 180px, ' +
+            'url(/assets/home-bg-left-z4.svg) no-repeat 0 100%/auto 295px, ' +
+            'url(/assets/home-bg-right-z3.svg) no-repeat 100% 100%/auto 400px',
+      backgroundAttachment: 'fixed !important',
```

Comment:
I'm not proud of the `!important` here, but there's a style rule that's overriding `background-attachment` higher up in the cascade somewhere.

Reply:
i couldn't find the overriding class either.. maybe some default behavior.

Reply:
Thanks for looking @karthikbtf 🙌